### PR TITLE
Release/6.1.0 rc.1

### PR DESCRIPTION
**Pull request by ethan605**
# Changelog

- Public: Introduced new system for locale keys. Keys are now more structured and easier to find within the code.

## Checklist

_put `n/a` if item is not relevant to PR changes_

- [x] Has the CHANGELOG been updated?
- [n/a] Has the README been updated?
- [n/a] Has the CONTRIBUTING doc been updated?
- [n/a] Has the RELEASE_GUIDELINES been updated?
- [n/a] Has the MIGRATION doc been updated for any MAJOR breaking changes?
- [n/a] Has the MIGRATION doc been updated for any MINOR breaking changes, including any translation strings or keys changes?
- [n/a] Have any new automated tests been implemented or the existing ones changed?
- [n/a] Have any new manual tests been written down or the existing ones changed?
- [n/a] Have any new strings been translated or the existing ones changed?

### CODE REVIEW

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-1214-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-1214-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://1214-pr-onfido-sdk-ui-onfido.surge.sh

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,32 @@ This change log file is based on best practices from [Keep a Changelog](http://k
...
+
+### Fixed
+
+- UI: Accessibility - Error and warning alert heading is now ARIA heading level 1
+- UI: Camera inactivity timeout only starts from camera access granted instead of on initial render
+- UI: Fixed call to action buttons covering content and instructions on Passport Image Guide, Selfie Intro screens when viewed on a mobile device with a shorter viewport, e.g. iPhone SE (1st gen)
+
 ## [6.0.1] - 2020-10-09
 
 ### Fixed
```
Comment (kopijunkie) [inline CHANGELOG.md:37]: The merge of #1215 seems to have created duplicate `6.0.1` version `CHANGELOG` entries

File: README.md [hunk 2]
```diff
@@ -350,7 +350,7 @@ A number of options are available to allow you to customise the SDK:
...
 
   - `phrases` (required) : An object containing the keys you want to override and the new values. The keys can be found in [`src/locales/en_US/en_US.json`](src/locales/en_US/en_US.json). They can be passed as a nested object or as a string using the dot notation for nested values. See the examples below.
-  - `mobilePhrases` (optional) : An object containing the keys you want to override and the new values. The values specified within this object are only visible on mobile devices. Please refer to the `mobilePhrases` property in [`src/locales/en_US/en_US.json`](src/locales/en_US/en_US.json).
+  - `mobilePhrases` (optional) : An object containing the keys you want to override and the new values. The values specified within this object are only visible on mobile devices. Please refer to the `mobilePhrases` property in [`src/locales/en_US/en_US.json`](src/locales/en_US/en_US.json). **Notes**: supports for standalone `mobilePhrases` key will be deprecated soon. Consider nesting it inside `phrases` if applicable.
```
Comment (kopijunkie) [inline README.md:353]:
```suggestion
  - `mobilePhrases` (optional) : An object containing the keys you want to override and the new values. The values specified within this object are only visible on mobile devices. Please refer to the `mobilePhrases` property in [`src/locales/en_US/en_US.json`](src/locales/en_US/en_US.json). **Note**: support for standalone `mobilePhrases` key will be deprecated soon. Consider nesting it inside `phrases` if applicable.
```

File: test/config.json [hunk 3]
```diff
@@ -9,20 +9,6 @@
...
-      "browsers": [
-        {
-          "browserName": "IE",
-          "browser_version": "11.0",
-          "os": "Windows",
-          "os_version": "10",
-          "resolution": "1920x1080",
-          "remote": true
-        }
-      ]
```
Comment (kopijunkie) [inline test/config.json:25]: Didn't notice it in previous PR(s), but why did we remove the IE11 UI tests? 🤔

**Review by kopijunkie — CHANGES_REQUESTED** [review]
`CHANGELOG` has ended up in a slightly weird state & IE11 UI tests seem to have been disabled. The rest LGTM.

File: CHANGELOG.md [hunk 4]
```diff
@@ -20,8 +46,9 @@ This project adheres to the Node [default version scheme](https://docs.npmjs.com
 - UI: Add country selection screen after document selection. This screen is skipped by default for a preselected document but can still be displayed by enabling the `showCountrySelection` option for the `document` step.
 - UI: New warnings for cut-off & blurry images detection.
 - UI: When the uploaded image is either cut-off, glary or blurry, the end-user must retry at most 2 times prior to proceeding further.
-- Internal: The release script and the `release/RELEASE_GUIDELINE.md` file now include the information needed to update the `MIGRATION.md` file.
 - UI: Added Residence Permit option for document selection
+- Internal: The release script and the `release/RELEASE_GUIDELINE.md` file now include the information needed to update the `MIGRATION.md` file.
+- Internal: Send additional `system` data in `sdk_metadata` which contains `os`, `os_version`, `browser` & `browser_version` info of the current session.
```
Comment (stefaniacardenas) [inline CHANGELOG.md:51]: Was this part of version `6.0.0`?

Comment (stefaniacardenas) [reply CHANGELOG.md:51]: It wasn't part of the changelog when the release was made https://github.com/onfido/onfido-sdk-ui/releases

**Review by stefaniacardenas — APPROVED** [review]

File: test/config.json (hunk 3 above)
Comment (ethan605) [reply test/config.json:25]: I think that's a mistake. I'll enable them again

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-1214-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-1214-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://1214-pr-onfido-sdk-ui-onfido.surge.sh

**Review by kopijunkie — CHANGES_REQUESTED** [review]
The `CHANGELOG` is still a bit mixed up regarding what was in version [6.0.1](https://github.com/onfido/onfido-sdk-ui/releases/tag/6.0.1) vs this release `6.1.0`

**Comment by ethan605** [comment]
> The `CHANGELOG` is still a bit mixed up regarding what was in version [6.0.1](https://github.com/onfido/onfido-sdk-ui/releases/tag/6.0.1) vs this release `6.1.0`

I paraphrased to reflect old changes [here](https://github.com/onfido/onfido-sdk-ui/releases)

**Review by kopijunkie — APPROVED** [review]

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-1214-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-1214-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://1214-pr-onfido-sdk-ui-onfido.surge.sh

**Pull request merged**